ui/apps/platform/src/Containers/VulnMgmt/Reports/Components/ScopeName.tsx

```diff
@@ -1,64 +1,63 @@
 import React, { ReactElement } from 'react';
```

Comment:
The main structural change in this PR is that the report scope details request was moved from this child component, that was solely used to display the scope name, to being immediately after the request that fetches the main scope details.

This allows us to detect a problem early and display the appropriate error messages outside the `ScopeName` component and then pass that data down to child components without requiring an additional request.